src/app/content/hooks/storeTextSize.ts

```diff
@@ -18,8 +18,10 @@ export const loadStoredTextSize = (services: MiddlewareAPI & AppServices) => asy
     return;
   }
 
-  if (typeof window !== 'undefined' && window.localStorage) {
-    storedTextSize = window.localStorage.getItem(textResizerStorageKey);
+  try {
+    storedTextSize = window?.localStorage.getItem(textResizerStorageKey);
```

Comment:
pretty sure the `window?.` will throw in prerendering, you may need to keep that first part of the condition using `typeof window !== 'undefined'`

Reply:
If it throws, it will be caught, though, right? For that matter, I could leave the `?` out.
I ran the prerender locally, and it seemed ok to me, but it's possible I don't know what to look for.